### DISCUSSION TITLE
Line chart doesn't render dashed lines when `optimizeLargeData` is true

### DISCUSSION THREAD

**Discussion by kaelens** [microsoft/fluentui#32449] [Q&A] [answered]
Hello!

I noticed that the [stokeDasharray property is missing when optimizeLargeData](https://github.com/microsoft/fluentui/blob/59f6a1b9abca484752b2397cb2e7f205d205d71d/packages/react-charting/src/components/LineChart/LineChart.base.tsx#L694) is true and [_only_ when it is true](https://github.com/microsoft/fluentui/blob/59f6a1b9abca484752b2397cb2e7f205d205d71d/packages/react-charting/src/components/LineChart/LineChart.base.tsx#L963). 

Was this intended? Is there a way around this?

**Comment by khmakoto** [comment]
FYI @AtishayMsft

**Comment by kaelens** [comment]
Hi! Is there any update on this?

**Comment by AtishayMsft** [comment, marked as answer]
Hey @kaelens thanks for reaching out.
To get the dash array property you can consider setting optimize large data property to false.
This property was left out intentionally for large data use cases.

If you really need this property with large data, feel free to add it to the path element and we will be happy to merge your changes.

**Reply by kaelens** [reply]
Thanks so much!
